src/lib/features/personal-dashboard/personal-dashboard-controller.e2e.test.ts

```diff
@@ -217,3 +217,52 @@ test('should return personal dashboard project details', async () => {
         ],
     });
 });
+
+test('should return Unleash admins', async () => {
+    await loginUser('new_user@test.com');
+    const adminRoleId = 1;
+    const userService = app.services.userService;
+
+    const admin = await userService.createUser({
+        username: 'admin',
+        rootRole: adminRoleId,
+    });
+    const admin2 = await userService.createUser({
+        username: 'John',
+        name: 'John Admin',
+        rootRole: adminRoleId,
+    });
+
+    // service account that shouldn't be listed in the output. Service
+    // accounts are enterprise functionality, so there's no service to
+    // call here
+    const [{ id: serviceAdminId }] = await db
+        .rawDatabase('users')
+        .insert({
+            username: 'service_admin',
+            is_service: true,
+        })
+        .returning('*');
+    await app.services.accessService.setUserRootRole(
+        serviceAdminId,
+        adminRoleId,
+    );
+
+    const { body } = await app.request.get(`/api/admin/personal-dashboard`);
+
+    const admins = body.admins;
+    admins.sort((a, b) => a.id - b.id);
```

Comment:
I'm not a big fan of sorting in e2e/component tests. Is it changing between test runs?

Reply:
no idea. why don't you like sorting the data in tests like this? i'm thinking it'll keep it from breaking regardless of any changes that we do on the back end. We can leave it out, of course, but it'd be annoying if it suddenly broke later when we haven't specified the orde.r

Reply:
be151d1851dc0445d30a1b1193ac93fa647df85c